recipes/iperf/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "iperf" %}
+{% set version = "3.2" %}
+{% set sha256 = "cb20d3a33e07a3b45a49a358b044f4998f452ef9d1a8a5cbde476b6ab9e9b526" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/esnet/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: ./configure && make && make install 
```

Comment:
Seems the install prefix should be specified here. Curious as to how this could even be getting installed/packaged correctly. 😕